button/stories/button.stories.tsx

```diff
@@ -1,13 +1,50 @@
-import type { Meta } from '@storybook/react';
-import type { ButtonProps } from '@mantine/core';
-import { Button } from '@mantine/core';
+import type { Meta, StoryObj } from '@storybook/react';
+import { iconNames } from '../../icon/src/icon';
+import { Button, ButtonProps } from '../src/button';
 
-export default {
+const meta: Meta = {
   title: 'Buttons/Button',
   component: Button,
+};
+
+export default meta;
+
+export const Playground: StoryObj<ButtonProps> = {
+  argTypes: {
+    children: { type: { name: 'string', required: true }, control: 'text' },
+    variant: {
+      type: { name: 'string', required: true },
+      control: 'select',
+      options: ['primary', 'secondary'],
+    },
+    compact: { type: { name: 'boolean', required: true }, control: 'boolean' },
+    disabled: { type: { name: 'boolean', required: true }, control: 'boolean' },
+    iconPosition: {
+      type: { name: 'string', required: true },
+      options: ['none', 'left', 'right'],
+      control: { type: 'select' },
+    },
+    icon: {
+      type: { name: 'string', required: false },
+      options: [iconNames[0]],
+      control: {
+        type: 'select',
+        labels: {
+          undefined: 'none',
```

Comment:
minor issue, only affects the Storybook UI, but `undefined` is a default value defined below and it supposed to be set to `none` in the UI automatically, but for some reason it's not, maybe they special case `undefined` which is wrong